### PR TITLE
Feature/#105 injectable

### LINKED ISSUE #105: Move to injectable
Because when using kiwi we still need to register everything by hand. 

This is not always easy and can be easily forgotten.

For a long time, we have been looking for a Dagger alternative in dart. It seems that injectable is the best option outthere. 
https://pub.dev/packages/injectable
https://pub.dev/packages/injectable_generator

Kiwi can be removed from the flutter-template.

### PR DESCRIPTION
**Pull request by ikbendewilliam**
fixed #105

### CODE REVIEW

**Comment by ikbendewilliam** [comment]
Dummy is still to check.

**Review by NicolaChimerapps — APPROVED** [review]
Looks good

**Comment by NicolaChimerapps** [comment]
@ikbendewilliam did you check if the strip scripts still work?

File: lib/widget/provider/provider_widget.dart [hunk 1]
```diff
@@ -27,7 +27,7 @@ class ProviderWidget<T extends ChangeNotifier> extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return ChangeNotifierProvider<T>(
-      lazy: lazy,
+      lazy: false,
```
Comment (vanlooverenkoen) [inline lib/widget/provider/provider_widget.dart:30]: Why?

Comment (ikbendewilliam) [reply lib/widget/provider/provider_widget.dart]: forgot to reset this, was during some testing... It should be fixed now

**Review by vanlooverenkoen — APPROVED** [review]

**Pull request merged**